### DISCUSSION TITLE
1.12.2 simple voice chat mod support?

### DISCUSSION THREAD

**Discussion by rebarakuk** [djpadbit/Sound-Physics#89] [Q&A]
is it compatible with simple voice chat mod on 1.12.2?

Also, I'll install both mod and plugin of simple voice chat.

Should your mod be installed in mod bukkit like Catserver to use with sipmle voice chat? I've known it's client side

**Comment by gatoborrachon** [comment]
check this issue: https://github.com/djpadbit/Sound-Physics/issues/74
i think this issue was fixed, it is closed now, but i'm not sure, its been almost 2 years, wow
